src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -341,7 +341,7 @@ export function validateTiffAlignment(tiff: TiffLocation, allowedError = 0.015):
   return true;
 }
 
-export function getTileName(originX: number, originY: number, grid_size: number): string {
+export function getTileName(centerX: number, centerY: number, grid_size: number): string {
```

Comment:
This was correct with originX, originY, this function does not need a center point, if anything it should be just `x` and `y` as it can be any point within the tile.